### PR TITLE
Project 1: Hello Microverse.

### PR DESCRIPTION
**Pull request by Tumworobere**
Additional description about the project and its features.
- Understanding Git, and GitHub. 
- Understanding GitHub Flow.
- Understaanding linters and the structure and use of HTML & CSS

### CODE REVIEW

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
```markdown
Hi Tumworobere Tumworobere, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has failing linter checks on PR. Kindly fix the linter issues and make sure that all checks show green.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

Invalid Code Review Request **does not count** into the code reviews limit._
```

**Comment by Tumworobere** [comment]
> 
> 
> ```gfm
> Hi Tumworobere Tumworobere, 
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> ## Invalid Code Review Request
> 
> You have submitted a project, that has failing linter checks on PR. Kindly fix the linter issues and make sure that all checks show green.
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> --------
> 
> Invalid Code Review Request **does not count** into the code reviews limit._
> ```

Thanks @SajjadAhmad14 for taking time to review my project.Let me correct the linter errors and resubmit

File: style.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/* stylelint-disable */
```
Comment (carlos-ssh) [inline style.css]: - Enable your linters and fix issues.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @Tumworobere , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Tumworobere) [reply style.css]: Thanks @Carlos for taking your time to review my project.I have worked on the requested changes!

**Review by carlos-ssh — APPROVED** [review]
Hi @Tumworobere , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Tumworobere** [comment]
> 
> 
> Hi @Tumworobere ,
> 
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
> Congratulations! 🎉
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thanks @carlos-ssh !

**Pull request merged**